src/sgx/libpoet_bridge/enclave.cpp

```diff
@@ -532,8 +525,12 @@ namespace sawtooth {
         }
 
         void Enclave::Enclave_FinalizeWaitCertificate(
+            const char* inPrevWaitCertificate,
+            size_t inPrevWaitCertificateLen,
             const char* inPrevBlockId,
             size_t inPrevBlockIdLen,
+            const char* inPoetBlockId,
```

Comment:
Rename PoetBlockId to something more meaningful. Is it previous block wait certificate signature ?